### PR TITLE
Add logging when PerformUpkeep log is received

### PR DESCRIPTION
**Pull request by RyanRHall**

### CODE REVIEW

File: pkg/types/types.go [hunk 1]
```diff
@@ -57,9 +58,11 @@ type PerformLogProvider interface {
 }
 
 type PerformLog struct {
-	Key           UpkeepKey
-	TransmitBlock BlockKey
-	Confirmations int64
+	Key             UpkeepKey
+	TransmitBlock   BlockKey
+	Confirmations   int64
+	TransactionHash common.Hash
```
Comment (EasterTheBunny) [inline pkg/types/types.go:64]: One goal of the plugin is to not introduce any chain specific dependencies. Either make the transaction hash a native type or create a wrapping type for it.

**Review by EasterTheBunny — CHANGES_REQUESTED** [review]

File: pkg/types/types.go [hunk 2]
```diff
@@ -57,9 +57,11 @@ type PerformLogProvider interface {
...
 
 type PerformLog struct {
-	Key           UpkeepKey
-	TransmitBlock BlockKey
-	Confirmations int64
+	Key             UpkeepKey
+	TransmitBlock   BlockKey
+	Confirmations   int64
+	TransactionHash string
+	BlockNumber     int64
```
Comment (infiloop2) [inline pkg/types/types.go:64]: We also need to ensure these get populated.  @RyanRHall @EasterTheBunny 

Also is blockNumber the same as transmit block? do we need both?

Comment (RyanRHall) [reply pkg/types/types.go:64]: blockNumber should indicate the blocknumber that includes the tx which produced the logs - different from transmit block

Comment (infiloop2) [reply pkg/types/types.go:64]: I think that is the transmit block number. As far as i understand there is checkBlockNumber: block at which check was done, transmitBlockNumber: block at which transmit was confirmed. cc @EasterTheBunny to correct me

**Pull request merged**